Traceback (most recent call last):
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/home/haxor/mambaforge/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/haxor/mambaforge/lib/python3.10/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import pandas as pd
data = pd.read_table("data.dat", 
    header=None, 
    sep='\s+', 
    names=["step", "time", "X", "Y", "VX", "VY"])
data
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mParserError[0m                               Traceback (most recent call last)
Cell [0;32mIn[10], line 2[0m
[1;32m      1[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m
[0;32m----> 2[0m data [38;5;241m=[39m [43mpd[49m[38;5;241;43m.[39;49m[43mread_table[49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mdata.dat[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m
[1;32m      3[0m [43m    [49m[43mheader[49m[38;5;241;43m=[39;49m[38;5;28;43;01mNone[39;49;00m[43m,[49m[43m [49m
[1;32m      4[0m [43m    [49m[43msep[49m[38;5;241;43m=[39;49m[38;5;124;43m'[39;49m[38;5;124;43m\[39;49m[38;5;124;43ms+[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m
[1;32m      5[0m [43m    [49m[43mnames[49m[38;5;241;43m=[39;49m[43m[[49m[38;5;124;43m"[39;49m[38;5;124;43mstep[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mtime[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mX[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mY[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mVX[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mVY[39;49m[38;5;124;43m"[39;49m[43m][49m[43m)[49m
[1;32m      6[0m data

File [0;32m~/mambaforge/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1405[0m, in [0;36mread_table[0;34m(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, date_format, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, encoding_errors, dialect, on_bad_lines, delim_whitespace, low_memory, memory_map, float_precision, storage_options, dtype_backend)[0m
[1;32m   1392[0m kwds_defaults [38;5;241m=[39m _refine_defaults_read(
[1;32m   1393[0m     dialect,
[1;32m   1394[0m     delimiter,
[0;32m   (...)[0m
[1;32m   1401[0m     dtype_backend[38;5;241m=[39mdtype_backend,
[1;32m   1402[0m )
[1;32m   1403[0m kwds[38;5;241m.[39mupdate(kwds_defaults)
[0;32m-> 1405[0m [38;5;28;01mreturn[39;00m [43m_read[49m[43m([49m[43mfilepath_or_buffer[49m[43m,[49m[43m [49m[43mkwds[49m[43m)[49m

File [0;32m~/mambaforge/lib/python3.10/site-packages/pandas/io/parsers/readers.py:626[0m, in [0;36m_read[0;34m(filepath_or_buffer, kwds)[0m
[1;32m    623[0m     [38;5;28;01mreturn[39;00m parser
[1;32m    625[0m [38;5;28;01mwith[39;00m parser:
[0;32m--> 626[0m     [38;5;28;01mreturn[39;00m [43mparser[49m[38;5;241;43m.[39;49m[43mread[49m[43m([49m[43mnrows[49m[43m)[49m

File [0;32m~/mambaforge/lib/python3.10/site-packages/pandas/io/parsers/readers.py:1923[0m, in [0;36mTextFileReader.read[0;34m(self, nrows)[0m
[1;32m   1916[0m nrows [38;5;241m=[39m validate_integer([38;5;124m"[39m[38;5;124mnrows[39m[38;5;124m"[39m, nrows)
[1;32m   1917[0m [38;5;28;01mtry[39;00m:
[1;32m   1918[0m     [38;5;66;03m# error: "ParserBase" has no attribute "read"[39;00m
[1;32m   1919[0m     (
[1;32m   1920[0m         index,
[1;32m   1921[0m         columns,
[1;32m   1922[0m         col_dict,
[0;32m-> 1923[0m     ) [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_engine[49m[38;5;241;43m.[39;49m[43mread[49m[43m([49m[43m  [49m[38;5;66;43;03m# type: ignore[attr-defined][39;49;00m
[1;32m   1924[0m [43m        [49m[43mnrows[49m
[1;32m   1925[0m [43m    [49m[43m)[49m
[1;32m   1926[0m [38;5;28;01mexcept[39;00m [38;5;167;01mException[39;00m:
[1;32m   1927[0m     [38;5;28mself[39m[38;5;241m.[39mclose()

File [0;32m~/mambaforge/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py:234[0m, in [0;36mCParserWrapper.read[0;34m(self, nrows)[0m
[1;32m    232[0m [38;5;28;01mtry[39;00m:
[1;32m    233[0m     [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mlow_memory:
[0;32m--> 234[0m         chunks [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_reader[49m[38;5;241;43m.[39;49m[43mread_low_memory[49m[43m([49m[43mnrows[49m[43m)[49m
[1;32m    235[0m         [38;5;66;03m# destructive to chunks[39;00m
[1;32m    236[0m         data [38;5;241m=[39m _concatenate_chunks(chunks)

File [0;32mparsers.pyx:838[0m, in [0;36mpandas._libs.parsers.TextReader.read_low_memory[0;34m()[0m

File [0;32mparsers.pyx:905[0m, in [0;36mpandas._libs.parsers.TextReader._read_rows[0;34m()[0m

File [0;32mparsers.pyx:874[0m, in [0;36mpandas._libs.parsers.TextReader._tokenize_rows[0;34m()[0m

File [0;32mparsers.pyx:891[0m, in [0;36mpandas._libs.parsers.TextReader._check_tokenize_status[0;34m()[0m

File [0;32mparsers.pyx:2061[0m, in [0;36mpandas._libs.parsers.raise_parser_error[0;34m()[0m

[0;31mParserError[0m: Error tokenizing data. C error: Expected 6 fields in line 6, saw 10


